vnext/PropertySheets/External/Microsoft.ReactNative.Common.targets

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- 
+  Copyright (c) Microsoft Corporation.
+  Licensed under the MIT License.
+
+  This file will be consumed by all app and module projects (Desktop & UWP,
+  both inside and outside of this repo) that build on top of
+  Microsoft.ReactNative.
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" InitialTargets="ReactNativeWindowsValidateProps">
+   
+  <Target Name="ReactNativeWindowsValidateProps">
+    <Error Condition="'$(ReactNativeWindowsDirNotSet)' == 'true'" Text="'ReactNativeWindowsDir' should be set to the resolved location of the `react-native-windows` package. See https://aka.ms/ReactNativeWindowsDir." />
```

Comment:
looks like https://aka.ms/ReactNativeWindowsDir points to a specific blob.
Just checking if intentioanl or if you were trying more for an evergreen style approach for when we change the expression...

Reply:
Yeah, concern was that what we were referring to could change.